CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 
 ## Bug Fixes
 - Updated `int_quickbooks__expenses_union` to filter on account classifications rather than types. This will now correctly include 'Other Expense' and 'Cost of Goods Sold' account types as Expense accounts in the `quickbooks__expenses_sales_enhanced` model.
+- Updated `docs.md` to fix incorrect `{% enddocs %}` statement.
```

Comment:
Thanks for adding this. I'll actually want to make a new `v0.18.2` patch release to incorporate this fix. I'll merge this into a staging branch to make some additional updates and will make those changes.